models/jira__daily_issue_field_history.sql

```diff
@@ -74,18 +78,25 @@ joined as (
         calendar.issue_id
 
         {% if is_incremental() %}    
-            {% for col in pivot_data_columns if col.name|lower == 'components' and var('jira_using_components', True) %}
-            , coalesce(pivoted_daily_history.components, most_recent_data.components) as components
-            {% endfor %} 
-            {% for col in pivot_data_columns if col.name|lower not in ['issue_day_id', 'issue_id', 'valid_starting_on', 'components'] %} 
-            , coalesce(pivoted_daily_history.{{ col.name }}, most_recent_data.{{ col.name }}) as {{ col.name }}
+            {% for col in pivot_data_columns %}
+                {% if col.name|lower == 'components' and var('jira_using_components', True) %}
+                , coalesce(pivoted_daily_history.components, most_recent_data.components) as components
+
+                {% elif col.name|lower not in ['issue_day_id', 'issue_id', 'valid_starting_on', 'components'] %} 
+                , coalesce(pivoted_daily_history.{{ col.name }}, most_recent_data.{{ col.name }}) as {{ col.name }}
+
+                {% else %}{% endif %}
```

Comment:
Or is this specific and dependent on each conditional use case in the model?

Reply:
I had included them because I thought not having them was causing an error I was getting at the time, but I just tried taking all the `else`s out, and there are no errors, so yes I agree they're pointless and have removed!